src/error/specification

```diff
@@ -1258,3 +1258,27 @@ severity:warning
 ingroup:plugin
 module:gpgme
 macro:GPGME_INVALID_RECIPIENT
+
+number:201
+description:The given port was not withing a valid range
+severity:error
+ingroup:plugin
+module:network
+
+number:202
+description:Could not find service name for port declaration
+severity:error
+ingroup:plugin
+module:network
+
+number:203
+description:Could not use port as it is already in use
```

Comment:
Do we really need so many errors? Aren't the descriptions and the reasons somewhat overlapping?

Reply:
Should I just do an "Port error" and reference to the specific error in the reason metadata?

Reply:
You should generally redesign the errors to be more user friendly. Think about what applications and users care about.

Only if there are situations where applications want to distinguish one error from the other, the error number should be different.

Reply:
Alternatively we could also introduce some inheritance for error messages and something like `kdbContainsError (KDB *key, int whicherror)` (or similar for the higher-level API).

We can discuss this tomorrow if you already want to work on that topic. But maybe you should finish the tasks you started already before.

Reply:
I refactored the errors and are only using a single one which I did not see anywhere else in the specification file. I also reused another error number from the range plugin

Reply:
Error messages will be dealt with another issue https://github.com/ElektraInitiative/libelektra/issues/1789